Content.Server/Pinpointer/ServerPinpointerSystem.cs

```diff
@@ -31,6 +32,9 @@ private void OnLocateTarget(HyperspaceJumpCompletedEvent ev)
             // this code update ALL pinpointers in game
             foreach (var pinpointer in EntityQuery<PinpointerComponent>())
             {
+                if (pinpointer.Emagged)
+                    continue;
+
```

Comment:
This should be a separate component rather than checking an irreversible field every tick.

Reply:
Does the emagged one even work across ftl.